### DISCUSSION TITLE
Request to add BBC Micro via MESS lr-mame

### DISCUSSION THREAD

**Discussion by SuperBadger77** [351ELEC/351ELEC#957] [Ideas]
I posted this on the discord yesterday but raising it here as an request:

Hi, I raised a request last year (https://github.com/AmberELEC/AmberELEC/issues/220)  to see if it would be possible to add a BBC emulator using Beebem. Whilst that has not proved fruitful, I was going to raise another issue on Github but it said to check here first. I've been using Batocera v35/v36 on my RG552 and it runs both the BBC and Acorn Electron flawlessly using lr-mame. I've created pad2key files for all my old favourite games and it works fantastically - I can play all the games on the RG552 without a keyboard. So was going to ask if it would be possible to get the BBC added via lr-mame (MESS) to Amber Elec using the same method (https://wiki.batocera.org/systems:bbc)? In Batocera its not configured to run on RK3326 devices though (assume a perf issue (although I think it may work)). I'd be happy to share all my pad2key mapping files with everyone. Apologies if this has been asked before.

**Comment by dhwz** [comment]
Hi, if you can share your files with me I would give it a try.
